website/docs/how-to/how-to-clone-environment.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: How to clone an environment
+---
+
+:::info availability
+
+Environment cloning is planned to be released in **Unleash 4.18**.
+
+:::
+
+Environment cloning is a tool to enable an admin to duplicate an existing environment, including all feature toggles strategies and their state. See [_Cloning environments_](../user_guide/environments#cloning-environments) for more information.
+
+## Step 1: Navigate to the environments page {#step-1}
+
+Navigate to the _Environments_ page in the admin UI (available at the URL `/environments`). Use the navigation menu item "Configure" and select "Environments".
+
+![The admin UI navigation "Configure" submenu with the Environments item highlighted.](/img/clone-environment-1.png)
+
+## Step 2: Select an environment to clone {#step-2}
+
+Select an environment to clone. On the right side, open the actions submenu and select "Clone".
+
+![The "production" environment actions submenu with the Clone option highlighted.](/img/clone-environment-2.png)
+
+## Step 3: Fill in the clone environment form {#step-3}
+
+Give your new environment a name. The name must be unique and cannot be the same as the original environment. The name is pre-filled with a suggestion, but you can change it to whatever you like.
+
+Select an environment type, and which projects to clone the environment configuration in. You also have the option to keep the users permission for the new environment.
```

Comment:
Hmm, I wonder if this can be clarified a bit further 🤔

What happens if they don't keep permissions?
```suggestion
Select an environment type and select which projects should have their environment configuration cloned. Choose whether to keep the existing user permissions for the new environment or ... [what happens here?]
```

Reply:
Good point, you get a fresh environment with no RBAC overrides for create/update/delete strategy or turn on/off a toggle in that environment. Effectively only an admin or editor of the project can fiddle with the strategies/enable state for that environment